onnxruntime/core/optimizer/transformer_memcpy.cc

```diff
@@ -109,14 +116,17 @@ This transformer does not currently optimize copies between, e.g., two different
 
 bool TransformerMemcpyImpl::ModifyGraph(const KernelRegistryManager& kernel_registries) {
   bool modified = false;
+  InitializedTensorSet initializers_consumed;
   // find defs that require copy
   for (auto& node : graph_.Nodes()) {
     //don't need to do node placement here now, onnxruntime will do it according to registered kernels.
```

Comment:
this comment is confusing. I think it can be removed.

Reply:
I mean this line "//don't need to do node placement here now, onnxruntime will do it according to registered kernels." :).

Reply:
Sure, will remove this comment.